pandas/tests/internals/test_internals.py

```diff
@@ -286,29 +286,6 @@ def test_delete(self):
         with pytest.raises(Exception):
             newb.delete(3)
 
-    def test_split_block_at(self):
-
-        # with dup column support this method was taken out
-        # GH3679
-        pytest.skip("skipping for now")
```

Comment:
no problem removing, though his should in theory work now

Reply:
The test doesn't work period.  The function "split_block_at" doesn't exist anywhere.